databricks_cli/clusters/cli.py

```diff
@@ -331,6 +331,44 @@ def cluster_events_cli(api_client, cluster_id, start_time, end_time, order, even
         click.echo(tabulate(_cluster_events_to_table(events_json), tablefmt='plain'))
 
 
+@click.command(context_settings=CONTEXT_SETTINGS)
+@click.option('--cluster-id', cls=OneOfOption, one_of=CLUSTER_OPTIONS,
+              type=ClusterIdClickType(), default=None, help=ClusterIdClickType.help)
+@click.option('--cluster-name', cls=OneOfOption, one_of=CLUSTER_OPTIONS,
+              type=ClusterIdClickType(), default=None, help=ClusterIdClickType.help)
+@click.option('--local-port', type=click.INT, help="the local port")
+@debug_option
+@profile_option
+@eat_exceptions
+@provide_api_client
+def tunnel_cli(api_client, cluster_id, cluster_name, local_port):
+    """
+    Start a secure TCP tunnel to a cluster over Databricks' identity proxy.
+    """
+    if sys.version_info < (3, 6):
+        raise RuntimeError("The tunneling command is not supported on Python version < 3.6")
```

Comment:
Isn't the python version tied to the CLI version? If so can we give them instructions to change CLI versions

Reply:
That is true if the CLI is using only one python version. However, the current CLI version supports both python `2.7` and `3.6`, but the tunneling min. req is using `3.6`. So we need this check in case the user is using python2.

Reply:
i'm not sure what the timeline is for the cli to only support python3 onwards.